website/static/js/project.js

```diff
@@ -313,6 +313,10 @@ $(document).ready(function() {
         window.location = '/search/?q=(tags:"' + $(e.target).text().toString().trim()+ '")';
     });
 
+    $('#shareDropDownMenu li a').on('click', function(e) {
```

Comment:
This appears to be the same (excepting the UI selector string) as something in `filepage/index.js`, consider [DRYing](https://en.wikipedia.org/wiki/Don't_repeat_yourself) this up.